### PR TITLE
Revert grant/revoke on schema feature

### PR DESCRIPTION
**Pull request by shalinilohia50**
### Description

Revert grant/revoke on schema feature

### Issues Resolved

[List any issues this PR will resolve]

### Test Scenarios Covered ###
* **Use case based -**

* **Boundary conditions -**

* **Arbitrary inputs -**

* **Negative test cases -**

* **Minor version upgrade tests -**

* **Major version upgrade tests -**

* **Performance tests -**

* **Tooling impact -**

* **Client tests -**

### Check List
- [ ] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is under the terms of the Apache 2.0 and PostgreSQL licenses, and grant any person obtaining a copy of the contribution permission to relicense all or a portion of my contribution to the PostgreSQL License solely to contribute all or a portion of my contribution to the PostgreSQL open source project.

For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/babelfish-for-postgresql/babelfish_extensions/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: test/JDBC/upgrade/13_6/schedule [hunk 1]
```diff
@@ -334,5 +334,3 @@ BABEL-2999
 drop_index-before-15_6-or-16_2
 permission_restrictions_from_pg
 BABEL-730-before-15_6-or-16_1
-GRANT_SCHEMA
-babel-4517
```
Comment (HarshLunagariya) [inline test/JDBC/upgrade/13_6/schedule:338]: Not expected

File: test/JDBC/upgrade/13_9/schedule [hunk 2]
```diff
@@ -331,6 +331,3 @@ operator_binary_whitespace-before-15_6-or-16_2
 BABEL-2999
 drop_index-before-15_6-or-16_2
 permission_restrictions_from_pg
-BABEL-730-before-15_6-or-16_1
-GRANT_SCHEMA
-babel-4517
```
Comment (HarshLunagariya) [inline test/JDBC/upgrade/13_9/schedule]: Not expected, should only remove GRANT_SCHEMA

File: test/JDBC/upgrade/14_3/schedule [hunk 3]
```diff
@@ -350,5 +350,4 @@ BABEL-2999
 drop_index-before-15_6-or-16_2
 permission_restrictions_from_pg
 BABEL-730-before-15_6-or-16_1
-GRANT_SCHEMA
-babel-4517
```
Comment (HarshLunagariya) [inline test/JDBC/upgrade/14_3/schedule:354]: Not expected

File: test/JDBC/upgrade/14_5/schedule [hunk 4]
```diff
@@ -365,5 +365,3 @@ BABEL-2999
 drop_index-before-15_6-or-16_2
 permission_restrictions_from_pg
 BABEL-730-before-15_6-or-16_1
-GRANT_SCHEMA
-babel-4517
```
Comment (HarshLunagariya) [inline test/JDBC/upgrade/14_5/schedule:369]: Not expected

File: test/JDBC/upgrade/14_6/schedule [hunk 5]
```diff
@@ -400,5 +400,3 @@ BABEL-2999
 drop_index-before-15_6-or-16_2
 permission_restrictions_from_pg
 BABEL-730-before-15_6-or-16_1
-GRANT_SCHEMA
-babel-4517
```
Comment (HarshLunagariya) [inline test/JDBC/upgrade/14_6/schedule:404]: Not expected

File: test/JDBC/upgrade/15_4/schedule [hunk 6]
```diff
@@ -464,5 +464,3 @@ drop_index-before-15_6-or-16_2
 BABEL-4606
 permission_restrictions_from_pg
 BABEL-730-before-15_6-or-16_1
-GRANT_SCHEMA
-babel-4517
```
Comment (HarshLunagariya) [inline test/JDBC/upgrade/15_4/schedule:468]: Not expected

File: test/JDBC/upgrade/13_6/schedule (hunk 1 above)
Comment (thephantomthief) [inline test/JDBC/upgrade/13_6/schedule:338]: We are removing BABEL-4517 file from many schedule files. It seems unrelated to GRANT ON SCHEMA. Could you double check if that should be done?

Comment (shalinilohia50) [reply test/JDBC/upgrade/13_6/schedule:338]: Done. It should be there.

**Review by HarshLunagariya — CHANGES_REQUESTED** [review]

**Review by thephantomthief — APPROVED** [review]
LGTM

**Review by HarshLunagariya — APPROVED** [review]

**Review by jsudrik — APPROVED** [review]
Approved.
